gammagl/layers/conv/mgnni_m_iter.py

```diff
@@ -0,0 +1,108 @@
+import numpy as np
+import tensorlayerx as tlx
+import tensorlayerx.nn as nn
+
+from gammagl.utils import degree
+from gammagl.layers.conv import MessagePassing
+
+
+class MGNNI_m_iter(MessagePassing):
+    def __init__(self, m, k, threshold, max_iter, gamma, layer_norm=False):
+        super(MGNNI_m_iter, self).__init__()
+        self.F = nn.Parameter(tlx.convert_to_tensor(np.zeros((m, m)), dtype=tlx.float32))
+        self.layer_norm = layer_norm
+        self.gamma = gamma
+        self.k = k
+        self.max_iter = max_iter
+        self.threshold = threshold
+        self.f_solver = fwd_solver
+        self.b_solver = fwd_solver
+
+    def reset_parameters(self):
+        initor = tlx.initializers.Zeros()
+        self.F = self._get_weights("F", shape=self.F.shape, init=initor)
+
+    def _inner_func(self, Z, X, edge_index, edge_weight, num_nodes):
+        P = tlx.ops.transpose(Z)
+        ei = tlx.ops.convert_to_tensor(tlx.ops.convert_to_numpy(edge_index))
```

Comment:
Why do we need ``convert_to_numpy`` and then use ``convert_to_tensor``? I think this is unnecessary.

Reply:
Thank you for your feedback. Initially, my intention was to remove the gradient of that tensor by converting it to numpy and then back to a tensor. However, after your reminder, I realized that using convert_to_tensor() directly would suffice. I have made the necessary revisions to the code accordingly.